src/winter_onboarding_2021/fleet_management_service/views/user.clj

```diff
@@ -0,0 +1,15 @@
+(ns winter-onboarding-2021.fleet-management-service.views.user)
+
+
+(defn signup-form []
+  [:form {:action "/users/signup" :method "POST" :enctype "multipart/form-data"}
+   [:div {:class "mb-3" :id "email"}
```

Comment:
We should try to extract smaller components and reuse them where we can. I think here we can extract the input and label in to a component. 

It should also be possible to also make the form itself a component which takes a form spec and renders it.

Reply:
>It should also be possible to also make the form itself a component which takes a form spec and renders it.

What exactly do you mean by form spec? Will it be used to pre-fill the form for editing? or do you want to make a general form component?

cc: @yogirajh007